src/Repository/EventRepository.php

```diff
@@ -437,15 +442,21 @@ public function searchAllEvents(SearchParametersFilter $search): array
             $filterCategory = '';
         }
 
+        if ($search->getReferentEvents()) {
```

Comment:
Why not a ternary condition here :thinking: ?

Reply:
With simple `if` it's more readable.